tasks/ebpf.py

```diff
@@ -1,5 +1,9 @@
 from __future__ import annotations
 
+import shutil
+
+from tasks.libs.common.git import get_current_branch
+from tasks.libs.common.utils import get_commit_sha
```

Comment:
```suggestion
from tasks.libs.common.git import get_commit_sha, get_current_branch
```

`get_commit_sha` is in `utils` because it's imported if I'm not wrong

Reply:
Woops, nice catch :D